File: 1007.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var A = Number(lines.shift());
var B = Number(lines.shift());
var C = Number(lines.shift());
var D = Number(lines.shift());
console.log(`DIFERENCA = ${A*B-C*D}`);

File: 1007.java
import java.util.Scanner;
public class Main {
        public static void main(String[] args) {
                Scanner input = new Scanner(System.in);
                int diferenca = (input.nextInt() * input.nextInt()) - (input.nextInt() * input.nextInt());
                System.out.println("DIFERENCA = " + diferenca);
        }
}
